Providers/Modules/Plugins/Common/plugin/out_oms_diag.rb

```diff
@@ -0,0 +1,111 @@
+module Fluent
+
+  class OutputOMSDiag < BufferedOutput
+
+    Plugin.register_output('out_oms_diag', self)
```

Comment:
@lagalbra Actually I need to know if we are to support backward compatibility. As that would imply that this plugin code will be sent to every omsagent. This was actually intended to be a part of release, so maybe we should have a change to support this if omsagent is above a version.